### PR TITLE
✨ Add ERD generator

### PR DESCRIPTION
**Pull request by dankolbman**
Addresses #43 . Generates an erd from the current definition of the schema.

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ pytest==3.3.1
 pytest-cache==1.0
 pytest-cov==2.5.1
 pytest-pep8==1.0.6
+-e git+git@github.com:dankolbman/eralchemy.git@1058c12b828013144f861cb59f0e2c280a1a5ef6#egg=eralchemy
```
Comment (dankolbman) [inline dev-requirements.txt]: Waiting for [this](https://github.com/Alexis-benoist/eralchemy/pull/42) to merge.

**Review by znatty22 — CHANGES_REQUESTED** [review]
This is pretty cool 👍,  Only comment is that it looks like File is getting rendered because its in dataservice.api.common.models. Maybe we should remove that since it'll go into its own package later

**Comment by dankolbman** [comment]
Technically accurate to have File in there. As to whether that ever should have been commited is another question...

**Review by znatty22 — APPROVED** [review]

File: dev-requirements.txt [hunk 2]
```diff
@@ -2,3 +2,4 @@ pytest==3.3.1
 pytest-cache==1.0
 pytest-cov==2.5.1
 pytest-pep8==1.0.6
+-e git+git@github.com:Alexis-benoist/eralchemy.git#egg=eralchemy
```
Comment (dankolbman) [inline dev-requirements.txt:5]: Will want to change this to track the pypy package once a release has been cut.

**Review by parimalak — APPROVED** [review]

**Pull request merged**